### DISCUSSION TITLE
Anyone can help cause i don't know what is wrong?

### DISCUSSION THREAD

**Discussion by Desouler** [Flexget/Flexget#3947] [Q&A]
Hi everyone,

I had a config for ages and it was working fine. I noticed that the last 2-3 weeks it's not working correctly anymore ( the issue may be older, that is just when i noticed. One example task is below:

`TLMovies:
    priority: 10

    template:
      - movies
      - transmission-movies
      - send-email
      - exec-movies

    discover:
      what:
        - trakt_list:
            <<: *trakt-tl-movies
            strip_dates: yes
      from:
        - torrentleech:
            <<: *torrentleech_credentials

    convert_magnet: yes

    list_match:
      from:
        - trakt_list:
            <<: *trakt-tl-movies
            strip_dates: yes
      single_match: yes
      remove_on_match: no

    list_remove:
      - trakt_list:
          <<: *trakt-tl-movies
      - trakt_list:
          <<: *trakt-watchlist
      - trakt_list:
          <<: *trakt-bay-movies
    list_add:
      - trakt_list:
          <<: *trakt-watched

    cfscraper: yes`

The get list from trakt through the discovery works fine and brings results. But afterwards on all the other instances i get:

> "Could not retrieve list from trakt (HTTPSConnectionPool(host='api.trakt.tv' , port=443): Max retries exceeded with url: /users/Daidalus/lists/tl_movies/items?limit=1000&page=1 (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x788ccc2b66d0>: Failed to establish a new connection: [Errno -2] Name or service not known')))"

From the net i found for trakt api the limits are 1000 calls every 5 minutes so it cannot be over usage of the api.

Anyone has any idea at all what is going on? I managed to find my access token inside sqlite (assuming is not encrypted) but i don't have an api key to test through postman for example the GET call myself to see the error or check rate limiting.

Any help is welcomed.

Thanks in advance

**Comment by BrutuZ** [comment]
Have you tried reauthenticating Trakt on Flexget?

IIRC they had a snafu some time last year that invalidated sessions for some people too.

**Reply by Desouler** [reply]
yes i tried both refresh and re authenticate. Maybe its not clear from the post, but the first time on "discover" it works! It is bringing the list normally. Every other time afterwards, does not (on "list_match" "list_remove" and "list_add"). And in particular it throws the error on exactly the same list that worked on discovery ("tl_movies") with the same username/password/code.
